tests/test_core.py

```diff
@@ -121,48 +121,500 @@ def test_evaluator_v1():
+        assert evaluate(
+            "max(0.0001,[0]+[1]/(pow(log10(x),2)+[2])+[3]*exp(-1*([4]*((log10(x)-[5])*(log10(x)-[5])))))",
+            [x],
+            v,
+        ) == pytest.approx(
+            max(
+                0.0001,
+                v[0]
+                + v[1] / (math.pow(math.log(x) / math.log(10), 2) + v[2])
+                + v[3]
+                * math.exp(
+                    -1
+                    * v[4]
+                    * (math.log(x) / math.log(10) - v[5])
+                    * (math.log(x) / math.log(10) - v[5])
+                ),
+            ),
+            abs=1e-15,
+        )
+        assert evaluate(
+            "max(0.0001,[0]+[1]/(pow(log10(x),2)+[2])+[3]*exp(-[4]*(log10(x)-[5])*(log10(x)-[5])))",
+            [x],
+            v,
+        ) == max(
+            0.0001,
+            v[0]
+            + v[1] / (math.pow(math.log(x) / math.log(10), 2) + v[2])
+            + v[3]
+            * math.exp(
+                -v[4]
+                * (math.log(x) / math.log(10) - v[5])
+                * (math.log(x) / math.log(10) - v[5])
+            ),
+        )
+
+    v = [0.945459, 2.78658, 1.65054, -48.1061, 0.0287239, -10.8759]
+    for x in [425.92155818]:
+        assert evaluate("-[4]*(log10(x)-[5])*(log10(x)-[5])", [x], v) == -v[4] * (
+            math.log10(x) - v[5]
+        ) * (math.log10(x) - v[5])
+
+    v = [55, 2510, 0.997756, 1.000155, 0.979016, 0.001834, 0.982, -0.048, 1.250]
+    # FIXME 1.0111198935028287 != 0.9954776655511116
+    # assert evaluate("[2]*([3]*([4]+[5]*log(max([0],min([1],x))))*1./([6]+[7]*100./3.*(max(0.,1.03091-0.051154*pow(x,-0.154227))-max(0.,1.03091-0.051154*pow(208.,-0.154227)))+[8]*((1+0.04432-1.304*pow(max(30.,min(6500.,x)),-0.4624)+(0+1.724*log(max(30.,min(6500.,x))))/max(30.,min(6500.,x)))-(1+0.04432-1.304*pow(208.,-0.4624)+(0+1.724*log(208.))/208.))))", [100.], v) == v[2] * (v[3] * (v[4] + v[5] * math.log(max(v[0], min(v[1], x)))) * 1. / (v[6] + v[7] * 100. / 3. * (max(0., 1.03091 - 0.051154 * math.pow(x, -0.154227)) - max(0., 1.03091 - 0.051154 * math.pow(208., -0.154227))) + v[8] * ((1 + 0.04432 - 1.304 * math.pow(max(30., min(6500., x)), -0.4624) + (0 + 1.724 * math.log(max(30., min(6500., x)))) / max(30., min(6500., x))) - (1 + 0.04432 - 1.304 * math.pow(208., -0.4624) + (0 + 1.724 * math.log(208.)) / 208.))))
+
+    assert evaluate("(1+0.04432+(1.724+100.))-1", [], []) == 101.76832
+    assert evaluate("(1+(1.724+100.)+0.04432)-1", [], []) == 101.76832
+    assert evaluate("((1.724+100.)+1+0.04432)-1", [], []) == 101.76832
+    # TODO: is this an important rounding issue?
+    assert evaluate("(1+0.04432+1.724/100.)-1", [], []) == pytest.approx(
+        0.06156, abs=1e-15
+    )
+    assert evaluate("(1+1.724/100.+0.04432)-1", [], []) == pytest.approx(
+        0.06156, abs=1e-15
+    )
+    assert evaluate("(1.724/100.+1+0.04432)-1", [], []) == pytest.approx(
+        0.06156, abs=1e-15
+    )
```

Comment:
@kirschen do you know the history of these checks? They seem unusually pedantic.

Reply:
Looking at the git blame history, looks like @dr15jones added them

Reply:
Ah the problem isn't with order of operations but rather float vs. double storage:
```
>>> (1+0.04432+1.724/100.)-1
0.06155999999999984
>>> (1+1.724/100.+0.04432)-1
0.06155999999999984
>>> (1.724/100.+1+0.04432)-1
0.06155999999999984
>>> 0.06156
0.06156
>>> 0.06156 - 0.06155999999999984
1.5959455978986625e-16
```